tests/symbol_dependent_transients_test.py

```diff
@@ -144,6 +172,27 @@ def test_symbol_dependent_pinned_array():
     assert (np.allclose(B, B_ref))
 
 
+@pytest.mark.skip  # @pytest.mark.gpu
+def test_symbol_dependent_gpu_view():
+    # NOTE: This test cannot produce the correct result since the input
+    # data of the reduction are not contiguous and cub:reduce doesn't support
```

Comment:
can't we force expansion to be pure here? we do this in some of the library states with a `with` statement

Reply:
Maybe the test should just change. It is not that important at this time.